po/tr.po

```diff
@@ -3220,193 +3228,235 @@ msgid ""
 "will this function panic?"
 msgstr ""
+"N'inci Fibonacci sayısını hesaplayan bir `fib(n)` fonksiyonu yazın. Bu "
+"fonksiyon ne zaman paniğe kapılacak?"
 
 #: src/types-and-values/exercise.md
 msgid "// The base case.\n"
-msgstr ""
+msgstr "// Temel durum.\n"
 
 #: src/types-and-values/exercise.md src/control-flow-basics/exercise.md
 msgid "\"Implement this\""
-msgstr ""
+msgstr "\"Bunu gerçekleştirin (implement)\""
 
 #: src/types-and-values/exercise.md
 msgid "// The recursive case.\n"
-msgstr ""
+msgstr "// Özyinelemeli durum.\n"
 
 #: src/types-and-values/exercise.md src/types-and-values/solution.md
-msgid "\"fib(n) = {}\""
-msgstr ""
+msgid "\"fib({n}) = {}\""
+msgstr "\"fib({n}) = {}\""
 
 #: src/control-flow-basics.md
 msgid "[if Expressions](./control-flow-basics/if.md) (4 minutes)"
-msgstr ""
+msgstr "[if İfadeleri](./control-flow-basics/if.md) (4 dakika)"
 
 #: src/control-flow-basics.md
 msgid "[Loops](./control-flow-basics/loops.md) (5 minutes)"
-msgstr ""
+msgstr "[Döngüler](./control-flow-basics/loops.md) (5 dakika)"
 
 #: src/control-flow-basics.md
 msgid ""
 "[break and continue](./control-flow-basics/break-continue.md) (4 minutes)"
 msgstr ""
+"[break ve continue](./control-flow-basics/break-continue.md) (4 dakika)"
 
 #: src/control-flow-basics.md
 msgid ""
 "[Blocks and Scopes](./control-flow-basics/blocks-and-scopes.md) (5 minutes)"
 msgstr ""
+"[Bloklar ve Kapsamlar](./control-flow-basics/blocks-and-scopes.md) (5 dakika)"
 
 #: src/control-flow-basics.md
 msgid "[Functions](./control-flow-basics/functions.md) (3 minutes)"
-msgstr ""
+msgstr "[Fonksiyonlar](./control-flow-basics/functions.md) (3 dakika)"
 
 #: src/control-flow-basics.md
 msgid "[Macros](./control-flow-basics/macros.md) (2 minutes)"
-msgstr ""
+msgstr "[Makrolar](./control-flow-basics/macros.md) (2 dakika)"
 
 #: src/control-flow-basics.md
 msgid ""
 "[Exercise: Collatz Sequence](./control-flow-basics/exercise.md) (15 minutes)"
 msgstr ""
-
-#: src/control-flow-basics.md src/generics.md src/modules.md
-msgid "This segment should take about 40 minutes"
-msgstr ""
+"[Alıştırma: Collatz Sekansı](./control-flow-basics/exercise.md) (15 dakika)"
 
 #: src/control-flow-basics/if.md
 msgid "`if` expressions"
-msgstr ""
+msgstr "`if` ifadeleri"
 
 #: src/control-flow-basics/if.md
 msgid ""
 "You use [`if` expressions](https://doc.rust-lang.org/reference/expressions/"
 "if-expr.html#if-expressions) exactly like `if` statements in other languages:"
 msgstr ""
+"[`if` ifadelerini](https://doc.rust-lang.org/reference/expressions/if-expr."
+"html#if-expressions) tam olarak diğer dillerdeki `if` deyimleri (statement) "
+"gibi kullanabilirsiniz:"
 
 #: src/control-flow-basics/if.md
 msgid "\"zero!\""
-msgstr ""
+msgstr "\"sıfır!\""
 
 #: src/control-flow-basics/if.md
 msgid "\"biggish\""
-msgstr ""
+msgstr "\"büyükçe\""
 
 #: src/control-flow-basics/if.md
 msgid "\"huge\""
-msgstr ""
+msgstr "\"büyük\""
```

Comment:
This probably shouldn't be the same as the translation for 'large'. Consider 'kocaman'.

Reply:
Thank you. Yes, it shouldn't be the same as "large" but I couldn't find a nice Turkish word instead. The word "kocaman" doesn't sound good. Maybe I could choose words like "dev" or "devasa". I need to consult a few people about this. Let me make a change based on the answers I get. I will make changes as soon as possible according to the answers I receive.